src/components/Controls.vue

```diff
@@ -77,6 +77,12 @@ export default defineComponent({
       return Object.values(nodeColumnTypes).map((colTypes) => Object.entries(colTypes).filter(([_, colType]) => colType === 'category').map(([varName, _]) => varName)).flat();
     });
     const maxConnections = computed(() => store.state.maxConnections);
+    const maxDegree = computed(() => store.state.maxDegree);
+    const degreeRange = ref([0, maxDegree.value]);
+
+    watch([maxDegree], () => {
+      degreeRange.value = [0, maxDegree.value];
+    });
```

Comment:
Can this be made into a computed? I'm not sure if this will have issues with reactivity. It could also cause issues with the value in the html below.

```suggestion
    const degreeRange = computed(() => [0, maxDegree.value]);
```